### ISSUE TITLE
cd_useraccountbank.xml

### ISSUE THREAD

**Issue by taaehk** [maisarah714/SimpleBank#3]
<CATALOG>
<CD>
<ID>123522</ID>
<TITLE>Mr</TITLE>
<NAME>ARIF HOTEL</NAME>
<POSITION>Business</POSITION>
<SALARY>RM900,000</SALARY>
</CD>
<CD>
<ID>123544</ID>
<TITLE>MRS</TITLE>
<NAME>PENULIS</NAME>
<POSITION>Business</POSITION>
<SALARY>RM1,000,000</SALARY>
</CD>
<CD>
<ID>123558</ID>
<TITLE>MR</TITLE>
<NAME>TAAEHK</NAME>
<POSITION>PENULIS</POSITION>
<SALARY>RM5,000,000</SALARY>
</CD>
</CATALOG>

**Comment by taaehk** [comment]
> 123522 <TITLE>Mr</TITLE> ARIF HOTEL Business RM900,000 123544 <TITLE>MRS</TITLE> PENULIS Business RM1,000,000 123558 <TITLE>MR</TITLE> TAAEHK PENULIS RM5,000,000

Create that as html


<!DOCTYPE html>
<html>
<style>
table,th,td {
  border : 1px solid black;
  border-collapse: collapse;
}
th,td {
  padding: 5px;
}
</style>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
<script>
$(document).ready(function(){
  $("#myInput").on("keyup", function() {
    var value = $(this).val().toLowerCase();
    $("#myTable tr").filter(function() {
      $(this).toggle($(this).text().toLowerCase().indexOf(value) > -1)
    });
  });
});
</script>
<body>
<p><center><input id="myInput" type="text" placeholder="Search from Bank Account . . ."></center></p>
<p><center><button type="button" onclick="loadXMLDoc()">User Bank Account Info</button></center></p>
<br><br>
<table id="demo"></table>

<script>
function loadXMLDoc() {
  var xmlhttp = new XMLHttpRequest();
  xmlhttp.onreadystatechange = function() {
    if (this.readyState == 4 && this.status == 200) {
      myFunction(this);
    }
  };
  xmlhttp.open("GET", "cd_bank.xml", true);
  xmlhttp.send();
}
function myFunction(xml) {
  var i;
  var xmlDoc = xml.responseXML;
  var table="<tr><th>ID</th><th>NAME</th><th>POSITION</th><th>SALARY</th></tr>";
  var x = xmlDoc.getElementsByTagName("CD");
  for (i = 0; i <x.length; i++) { 
    table += "<tr><td>" +   
    x[i].getElementsByTagName("ID")[0].childNodes[0].nodeValue +
    "</td><td>" +
    x[i].getElementsByTagName("NAME")[0].childNodes[0].nodeValue +
    "</td><td>" +
	x[i].getElementsByTagName("POSITION")[0].childNodes[0].nodeValue +
    "</td><td>" +
	x[i].getElementsByTagName("SALARY")[0].childNodes[0].nodeValue +
    "</td></tr>";
  }
  document.getElementById("demo").innerHTML = table;
}
</script>

</body>
</html>

**Comment by taaehk** [comment]
[ARIFHOTEL BANK SDN.BHD.zip](https://github.com/maisarah714/SimpleBank/files/4624339/ARIFHOTEL.BANK.SDN.BHD.zip)

haiz, suka kerja alang-alang
XD
